CanvasPlusPlayground/CourseAnnouncementManager.swift

```diff
@@ -18,8 +18,12 @@ import Foundation
     }
     
     func fetchAnnouncements() async {
+        
+        let endDate = Date.now
+        let startDate = endDate.addingTimeInterval(-1 * 60 * 60 * 24 * 30 * 5)
+        
         let announcements: [Announcement]? = try? await CanvasService.shared.defaultAndFetch(
-            .getAnnouncements(courseId: courseId),
+            .getAnnouncements(courseId: courseId, startDate: startDate.ISO8601Format(), endDate: endDate.ISO8601Format()),
```

Comment:
@iWatger can we just provide a Date object as a param, then you can do this conversion (`addingTimeInterval` + `ISO8601Format`) in Canvas Request under `queryParams`?

Reply:
28e653bfb130b6a09e728b2b1e155b79553f139f